hawkbit-rest/hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtTargetResource.java

```diff
@@ -156,6 +157,12 @@
         return ResponseEntity.ok().build();
     }
 
+    @Override
+    public ResponseEntity<Void> unassignTargetType(String targetId) {
+        this.targetManagement.unAssignType(targetId);
+        return ResponseEntity.ok().build();
+    }
+
```

Comment:
Same as within its interface `MgmtTargetRestApi` you should also provide an assign implementation.